docs/usage/TemplateExecutors.md

```diff
@@ -39,6 +39,25 @@ In addition to the `sprig` functions, landscaper specific functions are offered:
    type: ociRegistry
    imageReference: host:5000/myrepo/myimage:1.0.0
    ```
+- __generateImageOverwrite(componentDescriptor,componentDescriptorList): ImageVector__: returns the image vector 
```

Comment:
can you also add it to the Spiff docs.
I know there are more functions missing. But lets start with this function

Reply:
We have added some spiff docu